### DISCUSSION TITLE
Spinners causes frame drops when frame limiter is set above vsync

### DISCUSSION THREAD

**Discussion by felipemarins** [ppy/osu#22981] [Q&A]
Recently I tried to change the frame limiter from "VSync" to "2x refresh rate" because the mouse cursor felt slippier on vsync, but I noticed a huge frame drop everytime a spinner appears.
As my old notebook doesn't seem to allow me to record a smooth video to show the issue, I tried to take a screenshot:
![image](https://user-images.githubusercontent.com/53406701/227797098-426ecc1a-02e3-406f-afc2-037eef2ef753.png)
I don't know if this problem only occur to me because of my old hardware, so here is my specifications:
**OS:** Manjaro Linux
**CPU:** Intel Core i5-3230M
**GPU:** Intel HD Graphics 4000
**RAM:** 8GB

[database.log](https://github.com/ppy/osu/files/11072692/database.log)
[input.log](https://github.com/ppy/osu/files/11072693/input.log)
[network.log](https://github.com/ppy/osu/files/11072694/network.log)
[performance.log](https://github.com/ppy/osu/files/11072695/performance.log)
[performance-draw.log](https://github.com/ppy/osu/files/11072696/performance-draw.log)
[performance-update.log](https://github.com/ppy/osu/files/11072697/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/11072698/runtime.log)

Edit: I also tried on the "OpenGL (Legacy)" renderer and the problem persists. I remember experiencing this before the last update, but I ignored it at the time.

**Comment by peppy** [comment]
This could be due to the glow rendering or something else, hard to say. For the record, I've confirmed that this isn't the same issue as https://github.com/ppy/osu/issues/22985 and also cannot repro any stutters here. Will need to test with an HD4000 equiv I think.

**Comment by MineTurtlee** [comment]
Edit: Try to turn off UI (Shift + tab) and see. Mine is reduced lag now.
Rendering issues with glowing things and whatever, UI? I literally got no lags but my laptop.

**Reply by felipemarins** [reply]
Thanks, it resolved the lag issue to me.

**Reply by MineTurtlee** [reply]
No problems!
